src/components/TransactionProvider.css

```
@@ -0,0 +1,61 @@
+    transform 0.3s ease,
+    opacity 0.3s ease;
+  opacity: 1;
+  pointer-events: auto;
+}
+
+.notification.slide-in {
+  transform: translateY(0);
+  animation: slideIn 0.3s ease forwards;
+}
+
+@keyframes slideIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)